src/app/multiplayer/tic-tac-toe/page.js

```
@@ -0,0 +1,496 @@
+
+    return (
+        <>
+            {!gameStarted && (
+                <div className={styles.urlContainer}>{
+                    optionSelected ? (
+                        optionNewGame ? (
+                            <div>
+                                <div className={styles.urlWrapper}>
+                                    <p className={styles.urlText}>{currentGame === null ? 'Refresh the Page' : currentGame}</p>
+                                    <CopyToClipboard text={currentGame} onCopy={() => setStatus('Game number copied!')}>
+                                        <button className={styles.copyButton}>Copy Code</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)